setup/installation.md

```diff
@@ -46,39 +46,37 @@ kubectl config set-context --current --namespace=coder
    `helm search repo coder -l`)
 
    ```console
-   helm install coder coder/coder --namespace coder
+   helm install coder coder/coder -n coder --version=<VERSION>
```

Comment:
the long forms were intentional, because they're more readable than `-n` - can we change this back?

Reply:
I changed it to match the upgrade command below this, i'll change both though